Sources/SwiftParser/Declarations.swift

```diff
@@ -880,6 +888,11 @@ extension Parser {
     )
   }
 
+  mutating func atEnumCaseListTerminator() -> Bool {
+    return self.experimentalFeatures.contains(.trailingComma)
+      && (self.atStartOfLine || self.at(.rightBrace) || self.at(.semicolon))
+  }
+
```

Comment:
Wouldn’t this mean that we’d stop parsing the following as valid because we classify the comma as a trailing comma and then mark `bar` as unexpected?

```swift
enum A {
  case foo,
       bar
}
```

Similar to the where clause, I’m not sure if we want to allow a trailing comma here because there is no list terminator.

Reply:
Completely missed this, which is ironic because that's exactly how one would use a list with trailing comma 😅